Makefile

```diff
@@ -463,6 +463,12 @@ release-only:
 		echo 'Please update Added: tags in the documentation first.' ; \
 		exit 1 ; \
 	fi
+	@if $(grep -IPqrs '^>>>>>>> [0-9A-Fa-f]+' benchmark deps doc lib src test tools) || \
```

Comment:
Would it?  I'm thinking of situations where someone removes part of the conflict markers after manual resolution but not everything.